onecgiar-pr-server/src/api/ipsr/innovation-pathway/innovation-pathway-step-three.service.ts

```diff
@@ -745,6 +737,6 @@ export class InnovationPathwayStepThreeService {
   }
 
   isNullData(data: any) {
-    return data == undefined ? null : data;
+    return data ?? null;
```

Comment:
@JuanGuzman-io This is turning everything [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) into `null`. Is this correct?